azure-pipelines-scheduled.yaml

```diff
@@ -71,7 +71,7 @@ stages:
           - bash: |
               repo_sha=$(git rev-parse --verify HEAD)
               docker_image_tag_sha=${repo_sha:0:7}
-              last_commit_time=$(git log -1 --pretty='%cd' --date=iso | tr -d '+[:space:]:-' | head -c 14)
+              last_commit_time=$(date +'%Y%m%d%H%M%S')
```

Comment:
this was overriding the existing CI/CD tag as it uses the same repo_sha and with no new commits uses the same datetime.